.github/workflows/ci.yml

```diff
@@ -228,19 +146,14 @@ jobs:
       - name: Configure Sentry via the Editor Window
         run: dotnet msbuild /t:UnityConfigureSentryOptions /p:Configuration=Release /p:OutDir=other src/Sentry.Unity
 
+      # TODO: can we make unity only prepare the project, without compiling it, to unlock the unity license faster? iOS is already built like that.
```

Comment:
Yes please. Export gradle project.

Downside is that we don't cover some code path the one that uploads symbols when unity builds an apk. But we anyway have 2 paths to test

Reply:
We could do the other (=current) path later in another job later on. In any case I'd do this change in a followup PR.